paint-the-room/readme.md

```diff
@@ -0,0 +1,93 @@
+# Paint The Room 🧑‍🎨
+
+Estimating how long a task will take is a part of every job. In software development estimating how long a large task will take is difficult due to hidden complexity.
+
+Today we will be learning some effective techniques that allow us to still estimate tasks and plan through an interactive game.
+
+## Learning Objective 🧐
+
+We will be playing a game to learn important techniques to help us estimate our work:
+- Relative Estimation, 
+- Story Points, 
+- Story Elaboration, 
+- Story Splitting, 
+- Planning Poker, 
+- and Backlog Refinement 
+
+## Set up 🌼
+
+- Split into groups of no more than  5
+- You will need some paper and something to  write with.
+
+## The game 🎲
+
+Your team has been given the job of painting the room you are in. The customer’s vague requirement is: “Please paint the room.”
```

Comment:
Let's add a motivation here. Why do they want to know how to do this? How is this useful to them in their lives?

Reply:
I added motivation to the beginning with a real-world example and also a bit more background to the game. Let me know if you want me to switch it around anything 👍